camel/model_backend.py

```diff
@@ -66,7 +67,7 @@ def run(self, *args, **kwargs) -> Dict[str, Any]:
         num_max_token = num_max_token_map[self.model_type.value]
         num_max_completion_tokens = num_max_token - num_prompt_tokens
         self.model_config_dict['max_tokens'] = num_max_completion_tokens
```

Comment:
we also expose a util called `get_max_tokens()`  happy to expose this in this PR too